autocompletion/zsh/warden.plugin.zsh

```diff
@@ -0,0 +1,20 @@
+#compdef warden
+
+# Installation
+# ============
+#
+# Copy this file, `warden.plugin.zsh`, to `~/.oh-my-zsh/plugins/custom/warden/`
```

Comment:
maybe better to create a symlink? So if the file will update in next releases, it will be automatically updated there as well.